### DISCUSSION TITLE
Playback Every So Often Throws Error

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15317] [Ask A Question]
So this is happening a lot with the new 0.14 frigate. My wife was uses frigate to try to see when the baby fell asleep and every so often it throws random errors on playback. Sometimes it just spins forever, sometimes it throws an error like this. Is there any thing I can do to help prevent this?

Here is a sample of the error. There is nothing in the log files when this happens
![000044](https://github.com/user-attachments/assets/3a069305-34a6-47fe-9db6-83d17f9155eb)

**Comment by hawkeye217** [comment]
This error that Frigate displays is an indication that your Chromium-based browser itself is failing to decode an audio packet from your recording. This can be caused by a corrupted or damaged audio packet that came from your camera. Chromium-based browsers are more strict about decoding errors than other browsers.

For the best compatibility, make sure that AAC audio is coming from your camera. You could also try transcoding the audio to AAC with go2rtc (that may even be necessary if the AAC coming from your camera is corrupt in some way). Alternatively, try a non-Chromium based browser (Firefox or Edge, for example).

**Reply by MrCaspan** [reply]
All my Reolink cameras are AAC and I also use go2rtc to restream them all to make sure any issues are cleaned up.

I do have this in the config:

```
ffmpeg:
  input_args: preset-rtsp-restream
  output_args:
    record: preset-record-generic-audio-copy
```

as the stream is coming through in AAC I figure its better to use audio-copy thean re-encoding the audio. Is it worth maybe using `preset-record-generic-audio-aac` to re-encode the audio to ensure it's cleaned up?

**Reply by hawkeye217** [reply]
You could certainly try that.

**Reply by MrCaspan** [reply]
Ill give it a try and report back.. I know these Reolink camers seem to just have small problem after problem...

**Reply by hawkeye217** [reply]
Give a non-Chromium based browser a try as well.

**Reply by MrCaspan** [reply]
Question, why would bad audio stream just crash the entire stream. Why would it just isee the erro rbut ignore it, do you know? And is Edge not also Cromium?

**Reply by hawkeye217** [reply]
Sorry, yes, Edge is Chromium-based.

**Reply by MrCaspan** [reply]
So far it seems to be okay by doing the AAC re-encoding

**Reply by hawkeye217** [reply]
Great, thanks for letting us know.
